recordatorio_estudio.py

```diff
@@ -17,7 +17,13 @@ def center_window(window, width_percentage=0.25, height_percentage=0.25):
 def on_yes():
     def check_confirmation(event=None):
         if entry.get() == settings["message"][1]:
-            subprocess.run(rf'"{settings["exe_path"]}" --launch-product={settings["launch_product"]} --launch-patchline=live', shell=True)
+            cmd = rf'"{settings["exe_path"]}"'
+
+            if "extra_args" in settings.keys():
+                cmd = cmd + settings["extra_args"]
```

Comment:
sorry I missed this. There is a space needed bewteen cmd and its args. Possible fix:
`cmd = f"{cmd} {settings["extra_args"]}"`